prime-router/src/main/kotlin/Receiver.kt

```diff
@@ -115,7 +116,8 @@ open class Receiver(
         timeZone = timeZone,
         dateTimeFormat = dateTimeFormat,
         reverseTheQualityFilter = reverseTheQualityFilter,
-        enrichmentSchemaNames = enrichmentSchemaNames
+        enrichmentSchemaNames = enrichmentSchemaNames,
+        transport = transport
```

Comment:
Is this used?

Reply:
Yes used for setting the transport on a receiver to `NullTransport` so I can run the send step.

If I change the tests to not run the whole pipeline it will no longer be needed.